assets/state-mig-manager/0400_configmap.yaml

```diff
@@ -53,6 +53,78 @@ data:
           mig-devices:
             "7g.40gb": 1
 
+      # RTX-PRO-6000-96GB
+      all-1g.24gb+gfx:
+        - device-filter: ["0x2bb510DE"]
+          devices: all
+          mig-enabled: true
+          mig-devices:
+            "1g.24gb+gfx": 4
+
+      # RTX-PRO-6000-24GB
+      all-1g.24gb+me.all:
+        - device-filter: ["0x2bb510DE"]
+          devices: all
+          mig-enabled: true
+          mig-devices:
+            "1g.24gb+me.all": 1
+      
+      # RTX-PRO-6000-96GB
+      all-1g.24gb-me:
```

Comment:
I don't see a config for `all-1g.24gb`.

Reply:
Ah nvm, I see later on that a config already exists for it and we added the devid for RTX Pro 6000 there.